Sparkle/SUFileManager.m

```diff
@@ -507,13 +483,10 @@ - (BOOL)changeOwnerAndGroupOfItemAtRootURL:(NSURL *)targetURL toMatchURL:(NSURL
         return NO;
     }
     
-    NSString *formattedUserAndGroupIDs = [NSString stringWithFormat:@"%u:%u", ownerID.unsignedIntValue, groupID.unsignedIntValue];
-    char userAndGroup[PATH_MAX] = {0};
-    if (![formattedUserAndGroupIDs getFileSystemRepresentation:userAndGroup maxLength:sizeof(userAndGroup)]) {
-        if (error != NULL) {
-            *error = [NSError errorWithDomain:NSCocoaErrorDomain code:NSFormattingError userInfo:@{ NSLocalizedDescriptionKey: [NSString stringWithFormat:@"Owner ID %u and Group ID %u could not be formatted.", ownerID.unsignedIntValue, groupID.unsignedIntValue] }];
-        }
-        return NO;
+    char userAndGroup[100];
```

Comment:
Can zero-initilaize `= {0}` to be consistent with other code. Don't know about snprintf but some of C's string functions are dumb.